cloudify_nsx/network/dlr.py

```diff
@@ -62,6 +56,10 @@ def create(**kwargs):
         'router', kwargs, validation_rules
     )
 
+    print(router_dict)
+    router_dict = common.possibly_assign_vm_creation_props(router_dict)
+    print(router_dict)
```

Comment:
Replace to ctx.logger.info because in any case plugin will show parameters in get_properties_and_validate, so it will be not first place where we show dump. (Difference with previous line we changed properties so can show updated version.)